testsuite/features/upload_files/install/empty.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
```

Comment:
Can we wrap all these files in ../install/ folder on another folders organization like... `autoinstallation/cobbler/<operating_system>/<these_files>` ?

Reply:
Yes, autoinstall is a better name than install.